src/coord/src/catalog.rs

```diff
@@ -2165,8 +2176,12 @@ impl Catalog {
                         )
                         .map_err(|e| {
                             Error::new(ErrorKind::AmbiguousRename {
-                                depender: entry.name.to_string(),
-                                dependee: entry.name.to_string(),
+                                depender: self
+                                    .resolve_full_name(&entry.name, entry.conn_id())
```

Comment:
While we're here, might be nice to include a call to `minimal_qualification` to print the name as prettily as possible

Reply:
I agree that this would probably be beneficial, but `minimal_qualification` exists in `ConnCatalog`, and we only have access to a normal `Catalog`, so it would require some shuffling of APIs. Also there's a bunch more error messages/logs that should probably use `minimal_qualification` instead. So I might just punt on this until later, unless you feel strongly about it.